### DISCUSSION TITLE
Failed to upload csv [Apr 26]

### DISCUSSION THREAD

**Discussion by dstala** [nocodb/nocodb#1961] [Discord Threads]
> 🙋‍♂️ **Xpressd**  
#### any idea what this means? 😕  
![image0](https://cdn.discordapp.com/attachments/661905455894888493/967931188616507493/unknown.png)  
![image1](https://cdn.discordapp.com/attachments/661905455894888493/967931188834627584/unknown.png)

**Comment by o1lab** [comment]
> 👨‍🚀 **wingkwong**  
#### Not able to reproduce. please create the issue in github with some details.  
#  

 [Autogenerated from discord](https://discord.com/channels/661905455894888490/967931189719597096)  
- - - -  
#### Join [NocoDB's](http://www.nocodb.com/)  community  
[Discord](https://discord.gg/5RgZmkW) • [Twitter](https://twitter.com/nocodb) • [Reddit](https://www.reddit.com/r/NocoDB/) • [Docs](https://docs.nocodb.com/)
